src/deepstream.io.js

```diff
@@ -255,18 +257,11 @@ Deepstream.prototype._showStartLogo = function() {
 		return;
 	}
 	/* istanbul ignore next */
-	var logo =
-	' _____________________________________________________________________________\n'+
-	'                                                                              \n'+
-	'         /                                                             ,      \n'+
-	' ----__-/----__----__------__---__--_/_---)__----__----__---_--_-----------__-\n'+
-	'   /   /   /___) /___)   /   ) (_ ` /    /   ) /___) /   ) / /  )    /   /   )\n'+
-	' _(___/___(___ _(___ ___/___/_(__)_(_ __/_____(___ _(___(_/_/__/__o_/___(___/_\n'+
-	'                       /                                                      \n'+
-	'                      /                                                       \n'+
-	'=============================== STARTING... ==================================\n';
+	var logo = fs.readFileSync( './ascii-logo.txt', 'utf8' );
+
 	/* istanbul ignore next */
-	process.stdout.write( this._options.colors ? logo.yellow : logo );
+	process.stdout.write( logo + EOL );
+	process.stdout.write( ' =========================   ' + ( this._options.colors ? 'starting'.cyan : 'starting' ) + '   ==========================' + EOL );
 };
```

Comment:
Since the refactoring for the new logger API we dropped the color option at the root level.
Color can be set as a property of the logger options and it got renamed to `colorize`, this is because you can define multiple loggers and we're just passing the whole optoins object to the logger implementation which is winston currently.
The color was inconsistently anyway, because the setColor function was not part of the logger API.